streampark-flink/pom.xml

```diff
@@ -41,6 +41,7 @@
         <module>streampark-flink-proxy</module>
         <module>streampark-flink-packer</module>
         <module>streampark-flink-kubernetes</module>
+        <module>streampark-flink-repl</module>
```

Comment:
Hi @1996fanrui , thanks for your clarification.Streampark-flink-repl is now an experimental module that the original implementation would have depended on Checkstyle violation. Therefore, it is not added to Module list for the time being, and the whole package is passed
![image](https://user-images.githubusercontent.com/45427129/194497103-3833bb79-591d-4b53-937f-bcd754364691.png)